INTITA/Modules/ForgotPassword/ForgotPasswordViewController.swift

```diff
@@ -58,7 +56,7 @@ class ForgotPasswordViewController: UIViewController, Storyboarded {
         view.layoutSubviews()
         forgotTableView.endUpdates()
         
-        forgotTableView.scrollToRow(at: IndexPath(row: 3, section: 0), at: .bottom, animated: true)
+        forgotTableView.scrollToRow(at: IndexPath(row: 4, section: 0), at: .bottom, animated: true)
```

Comment:
Instead of `4` you may use `ForgotCells. sendButtonCell .rowIndex`
```
enum ForgotCells: Int {
     case logoImageCell = 1 // if possible please change back to 0
     case explanationLabelCell
     case emailTextFieldCell
     case emptyCell	
     case sendButtonCell
     
     var rowIndex: Int {
         return rawValue // -1 - take a look
     }
}
```

Reply:
I have just done it.